src/Symfony/Component/HttpKernel/Attribute/MapQueryString.php

```diff
@@ -29,6 +30,7 @@ public function __construct(
         public readonly array $serializationContext = [],
         public readonly string|GroupSequence|array|null $validationGroups = null,
         string $resolver = RequestPayloadValueResolver::class,
+        public readonly int $errorHttpStatusCode = Response::HTTP_NOT_FOUND,
```

Comment:
Personally I would rename this to `validationFailedCode` I think or `validationFailedStatusCode`? Same ofc. for the `MapRequestPayload` attribute.

Reply:
Agreed 😄

Reply:
Changed param name to validationFailedStatusCode